src/core/util/log.cc

```diff
@@ -131,19 +183,9 @@ void SetupLogging(const boost::program_options::variables_map& kovri_config)
   auto file_sink = boost::shared_ptr<text_file_sink>(
       std::make_unique<text_file_sink>(file_backend));
   // Set sink formatting
-  logging::formatter format =
-      expr::stream << "["
-                   << expr::format_date_time(
-                          expr::attr<boost::posix_time::ptime>("TimeStamp"),
-                          "%Y.%m.%d %T.%f")
-                   << "]"
-                   << " [" << expr::attr<attrs::current_thread_id::value_type>(
-                                  "ThreadID")
-                   << "]"
-                   << " [" << logging::trivial::severity << "]"
-                   << "  " << expr::smessage;
-  text_sink->set_formatter(format);
-  file_sink->set_formatter(format);
+  text_sink->set_formatter(
+      GetFormat(true && !kovri_config["log-disable-color"].as<bool>()));
```

Comment:
I think this line points out a problem elsewhere. We should have enabled-by-default options (I should've recommended that before, sorry). If we simply refactor to `--log-enable-color` and set default to true, then we can `GetFormat(kovri_config["log-disable-color"].as<bool>()`.  What do you think?